ifrs_report/__openerp__.py

```diff
@@ -33,6 +33,7 @@
     "depends": [
         "base",
         "account",
+        "account_group_auditory",
```

Comment:
@hbto auditory group account is in addons/vauxoo, and Odoo-ifrs is dependent because if you want to install only this module, Odoo-ifrs must know that module, since the acl are added to the group Auditor read-only which is in account_group_auditory